ebml/SafeReadIOCallback.h

```diff
@@ -77,7 +77,7 @@ class EBML_DLL_API SafeReadIOCallback {
   void Read(void *Dst, size_t Count);
 
   void Skip(size_t Count);
-  void Seek(size_t Position);
+  void Seek(size_t Offset);
```

Comment:
This is the one change I'd like to have the other way around. As `Seek` is only taking a single parameter, `Position` is the more appropriate term. Otherwise people might think wonder what the `Offset` is relative to. Lower-level file APIs such as POSIX `fseek` do have two parameters, one `Offset` and one to tell where to offset from (current position, start of the file, end of the file). Our API only has one parameter, hence `Position` being… somewhat better, I think.